content/kb/deployments/authentication-without-sso.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Authentication without SSO
+slug: /knowledge-base/deploy/authentication-without-sso
+---
+
+# Authentication without SSO
+
+## Introduction
+
+Want to secure your Streamlit app with passwords but Streamlit Cloud Teams and Enterprise doesn't support your SSO provider yet? We got you covered! This guide shows you two simple techniques for adding basic authentication to your Streamlit app, using [secrets management](/streamlit-cloud/get-started/deploy-an-app/connect-to-data-sources/secrets-management).
```

Comment:
By moving this to the knowledge base, we should probably revise this sentence to something like "Want to secure your Streamlit app with passwords, but cannot implement single sign-on?" This better reflects that community tier doesn't have the ability to do so, and Streamlit Cloud potentially doesn't support a specific provider

Reply:
Agreed! I've changed it to "Want to secure your Streamlit app with passwords, but cannot implement single sign-on?"